cfpb/consumerfinance.gov
# TCCP: Select multiple situations, placeholder situation content

This PR modifies the TCCP "Explore credit cards for your situation" landing page by allowing users to select more than one situation at a time. The situation content on the landing page has also been refreshed.

The results view now also shows situation content relevant to the selected situations. This content is placeholder and will be updated.

## How to test this PR

Run a local server and visit http://localhost:8000/consumer-tools/credit-cards/explore-cards/. Experiment with selecting one or more situations to see how the results view appears.

This branch is now also live on (internal) DEV4.

## Notes and todos

There are a few minor frontend tweaks that I could use help from @contolini. First, the icons on the select view don't exactly line up with the text as they should:

<img width="362" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/aacbf49a-de47-4d10-aae5-a75366c7b531">

I tried experimenting with a few CSS tweaks but couldn't figure this out. Additionally, I'm not super-confident in how I implemented the CSS classes so any feedback or improvements would be appreciated!

The content here also needs to be updated, although that can happen in a future PR.

## Checklist
